src/DataCollection.Shared/ViewModels/FeatureViewModel.cs

```diff
@@ -94,7 +96,9 @@ private async void UpdateImageSource()
                     return;
                 }
 
-                var symbol = await PopupManager.Symbol.CreateSwatchAsync(16, 16, 192, System.Drawing.Color.Transparent);
+                // Use different sizes depending on the geometry type to get best results.
```

Comment:
We've spoken about this because it feels a bit hacky and too customized to our dataset... are we sure this is the best method?  Should we not allow the SDK to derive the symbol based on its inherent size? I'm afraid of some downstream unintended consequences for symbols from other data authors whose sizes are meticulously calculated.

`public virtual Task<RuntimeImage> CreateSwatchAsync( double dpi = 96 )`